app/src/main/res/layout/home_activity.xml

```diff
@@ -1,8 +1,41 @@
-<?xml version="1.0" encoding="utf-8"?>
-<FrameLayout
-  xmlns:android="http://schemas.android.com/apk/res/android"
+<androidx.drawerlayout.widget.DrawerLayout xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto"
   xmlns:tools="http://schemas.android.com/tools"
-  android:id="@+id/home_fragment_placeholder"
+  android:id="@+id/drawer_layout"
   android:layout_width="match_parent"
   android:layout_height="match_parent"
-  tools:context=".home.HomeActivity" />
+  tools:context=".home.HomeActivity">
+
+  <LinearLayout
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical">
+
+    <LinearLayout
+      android:id="@+id/container_toolbar"
+      android:layout_width="match_parent"
+      android:layout_height="wrap_content"
+      android:orientation="vertical">
+
+      <include
+        android:id="@+id/toolbar"
+        layout="@layout/toolbar" />
+    </LinearLayout>
+
+    <FrameLayout
+      android:id="@+id/home_fragment_placeholder"
+      android:layout_width="match_parent"
+      android:layout_height="0dp"
+      android:layout_weight="1" />
+
+  </LinearLayout>
+  <fragment
```

Comment:
Ditto: Extra blank lines

Reply:
please refer doc for xml formatting guidelines 
https://docs.google.com/document/d/1AHIJagMfATgpbNCFmYoYaSqzsKp8JjTgPLQFXEFJdUA/edit?usp=sharing